### ISSUE TITLE
ARABIC VERSION

### ISSUE THREAD

**Issue by elfalehdev** [tunpl/tha9fni.tn#5]
We need to think of How we can display the content in arabic. Not mentioning that the side could be re:formated for this. We need to add a translate button works for multiple languages. OUR MAIN LANG IS ARABIC. So it all should be around that, renaming all the aspects asap.

**Comment by jagarkin** [comment]
Indeed, since Arabic is our main language, and I think this issue can be resolved easily in some way I can think of 
the **useContext**  hooks and of course it will be above the calling component in the tree and 3 json files will be provided 'en', 'fr' and 'Ar' where 'Ar' set to default and the value of chosen language will be stored into local storage.

or we can use the built in translate API of the browser which will allow translation of the content but I don't recommend this since most of people use 'fr' or 'en' as their display language.

so this probably might come last when content is already being written

**Comment by elfalehdev** [comment]
You're right. I think we can edit the core to be rounded in arabic. and we can integrate other languages later on.
